ApollosProject/apollos-apps
# feat: Community Single Navigation

## Motivation
Community work for the cycle is still being shaped out a bit, but we still wanted to begin making code progress. So this PR introduces the navigation and the Screen Component for what will be our Community Single

## How to Test
I've updated the Production dB to introduce a Community Tab. It holds a single card that is deep linked to the Community Single screen. You'll be able to use that in order to see this work in action!

If you want to test locally, just trigger a command (however you so desire) to link to: `/community/:communityId`

## Screenshots 
### New Community Tab
![Simulator Screen Shot - iPhone 13 - 2022-08-22 at 13 41 56](https://user-images.githubusercontent.com/45076058/185986573-5ed860da-fb5b-460b-8be9-cb78cf0c054c.png)

### Community Single in Action

https://user-images.githubusercontent.com/45076058/185987281-2e549e0b-5a81-4af3-8abf-0c7ff86b7fec.mp4

# NOTE :: this feature is not considered Production Ready, although it is protected from end-users since the route is hidden and only available through intentional integration
